### PR TITLE
IDEX- 2053 diff highlighting

### PR DESCRIPTION
**Pull request by mleduque**

### CODE REVIEW

File: codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java [hunk 1]
```diff
@@ -42,6 +44,9 @@ public HistoryAction(HistoryPresenter presenter,
     @Override
     public void actionPerformed(ActionEvent e) {
         eventLogger.log(this);
-        presenter.showDialog();
+        if (this.presenter == null) {
```
Comment (vparfonov) [inline codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java]: Why we still need field "presenter" if you use Provider ?

Comment (mleduque) [inline codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java]: Ah I hadn't seen that the HistoryPresenter was bound as a singleton.
So you're right, there's no need for that field.
Amending the PR now...

**Comment by mleduque** [comment]
Updated the pull request: remove 'presenter' field.

File: codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java (hunk 1 above)
Comment (mleduque) [inline codenvy-ext-git/src/main/java/com/codenvy/ide/ext/git/client/action/HistoryAction.java]: Updated, thanks.

**Pull request merged**